gamification/src/Components/Group/Dynamic/Teacher.js

```diff
@@ -1,13 +1,59 @@
 import React from 'react';
-import { Container, Row, Col } from 'react-bootstrap';
+import { Container, Row, Col, Button } from 'react-bootstrap';
+
+import StudentListItem from '../../StudentListItem/StudentListItem';
 
 function Teacher() {
+
+  const dummyStudents = [
+    {
+      id: 'A01374448',
+      name: 'Estudiante1',
+      hp: 0,
+      xp: 0,
+      gold: 0,
+      average:0.0
+    },
+    {
+      id: 'A01374866',
+      name: 'Estudiante2',
+      hp: 0,
+      xp: 0,
+      gold: 0,
+      average:0.0
+    },
+    {
+      id: 'A01374785',
+      name: 'Estudiante3',
+      hp: 0,
+      xp: 0,
+      gold: 0,
+      average:0.0
+    }
+  ];
+  const dummyStudentsList = dummyStudents.map(student => <StudentListItem key={student.id} student={student} />)
+
   return (
     // <div className="StudentGroup">
     <div>
       <Container>
```

Comment:
add tabs instead of containers

https://reactcommunity.org/react-tabs/
https://vaadin.com/components/vaadin-tabs/